portfolio/src/main/webapp/fetch.js

```diff
@@ -0,0 +1,8 @@
+
+async function showServletContent(){
+    const responseServer = await fetch ('/hello');
+    const textResponse = await responseServer.text();
+
+    const message = document.getElementById ('message-container');
```

Comment:
[nit] random space here in `getElementById ('message`? Same as with after `fetch (` above on L3 

I actually would have expected this to cause a syntax error when running, Javascript surprises me all the time 😮